lesson9-Logging/BrainstormSessions/Api/IdeasController.cs

```diff
@@ -99,15 +101,19 @@ public async Task<ActionResult<List<IdeaDTO>>> ForSessionActionResult(int sessio
         [ProducesResponseType(404)]
         public async Task<ActionResult<BrainstormSession>> CreateActionResult([FromBody]NewIdeaModel model)
         {
+            _logger.Debug($"Start of CreateActionResult Method execution with model {nameof(model)}");
```

Comment:
So the message will be `Start of CreateActionResult Method execution with model model`?
Cause nameof returns the name of the variable in this case.

Reply:
yes, we can think about a helper that allows us to print it in `Key: Value` view

Reply:
done